tools/travis.ps1

```diff
@@ -236,7 +235,7 @@ elseif($Stage -eq 'Build')
 
     # Running tests, which require sudo.
     $pesterParam['Tag'] = @('RequireSudoOnUnix')
-    $pesterParam['ExcludeTag'] = @()
+    $pesterParam['ExcludeTag'] = @('RequireNoSudoOnUnix')
```

Comment:
I don't think the `REQUIRENOSUDOONUNIX` tag is needed.
As you have fixed here, the problem is at line 217. The `ExcludeTag` was set to empty incorrectly when running without `SUDO`.

When running with `SUDO`, the `Tag` and `ExcludeTag` were set correctly -- only `Describe` blocks with the `RequireSudoOnUnix` tag will be included.

Reply:
Ok, will remove.